internal/util/helpers_test.go

```diff
@@ -4,18 +4,758 @@
 package util_test
 
 import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	"slices"
 	"strconv"
+	"strings"
+	"sync"
 	"testing"
+	"time"
 
 	_ "github.com/lib/pq"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
 
 	"github.com/mindersec/minder/internal/util"
+	pb "github.com/mindersec/minder/pkg/api/protobuf/go/minder/v1"
 )
 
+// TestGetConfigDirPath tests the GetConfigDirPath function
+func TestGetConfigDirPath(t *testing.T) {
+	t.Parallel()
+	var envLock sync.Mutex
```

Comment:
I think you want `envLock` to be at the file level, e.g. on line 32 before this function, because you have multiple tests all trying to set the environment, and you don't want any of them conflicting with each other.